### DISCUSSION TITLE
como posso ajudar aqui?

### DISCUSSION THREAD

**Discussion by ghost** [frontendbr/forum#2476] [Perguntas e Respostas] [answered]

**Comment by AlexsandroSA** [comment, marked as answer]
Sugestões de um usuário não tão ativo quanto gostaria. 

 - faça perguntas
 - responda perguntas
 - apioar projetos open source _(imagino que deva ter algo sobre isso)_ 
 
Espero ter ajudado. É isso que estou tentando fazer 
Até logo 👋🏽
